src/foam/nanos/notification/Notification.js

```diff
@@ -20,33 +20,33 @@ foam.CLASS({
     {
       class: 'Boolean',
       name: 'read',
+      documentation: 'Determines if notification has been read.',
       visibility: foam.u2.Visibility.RO
     },
     {
       class: 'Long',
       name: 'id',
       visibility: foam.u2.Visibility.RO
     },
-    {
```

Comment:
Why remove template property?

Reply:
I suspect this was a merge conflict resolution - do you remember?